x/gov/sonar-project.properties

```diff
@@ -0,0 +1,14 @@
+sonar.projectKey=cosmos-sdk-gov
```

Comment:
@julienrbrt this file?

Reply:
Yea, I somehow missed it 👍

Reply:
Created https://sonarcloud.io/project/configuration?id=cosmos-sdk-gov